rocketchat_API/APISections/integrations.py

```diff
@@ -0,0 +1,91 @@
+from rocketchat_API.APISections.base import RocketChatBase
+
+
+class RocketChatIntegrations(RocketChatBase):
+    # pylint: disable=too-many-arguments
+    def integrations_create(
+        self,
+        integrations_type,
+        name,
+        enabled,
+        username,
+        channel,
+        script_enabled,
+        event=None,
+        urls=None,
+        **kwargs
+    ):
+        """Creates an integration."""
+        if integrations_type == "webhook-outgoing":
```

Comment:
Is there any way we could test this path?

Reply:
Creation is tested below the incoming one : https://github.com/jadolg/rocketchat_API/blob/3c4cfb970baabfec5d9091f8335d0392fafe275e/tests/test_integrations.py#L22-L32

I did not bother with the others functions, I don't think they are relevant here (maybe the update one?) and add much value.

Reply:
Ok while posting I saw my mistake, I forgot to change the `integrations_type` :man_facepalming: 
I'll fix it.